source/glossary.rst

```diff
@@ -34,3 +34,6 @@ Glossary
         invited by a maintainer to oversee the development of one or multiple
         subsystems in a repository. More information
         :ref:`here <contrib_roles>`.
+
+    MUT
+        Module Under Test. More information :ref:`here <bao_tests>`.
```

Comment:
Try to give a proper definition to the term. Something as "...refers to the specific component of the system that is currently being subjected to testing to ensure its proper functionality and correctness. " Or something more clear and specific of the testing framework.

Reply:
Besides, a newline must be in-place between terms or this will trigger an error on the build.